### DISCUSSION TITLE
Rendering mantine in remix js not smooth

### DISCUSSION THREAD

**Discussion by mhanifmuhsin** [mantinedev/mantine#2770] [Q&A] [answered]
I have a problem rendering, here is the screen capture:
[https://www.loom.com/share/7fce8e55edda44a8b92e014860f20798](url)
Maybe there are friends who have experienced something like this, what should I check first ?

**Comment by rtivital** [comment, marked as answer]
You have broken ssr, follow remix getting started guide, use remix template as reference.

**Reply by mhanifmuhsin** [reply]
Thank's, resolved.
